code/parse/sexp/sexp_lookup.cpp

```diff
@@ -82,8 +83,22 @@ void parse_sexp_table(const char* filename) {
 	}
 }
 
+void free_lua_sexps(lua_State* /*L*/)
+{
+	// Remove all Lua sexps from our list so that there are no dangling pointers on the lua state
+	for (auto iter = operator_const_mapping.begin(); iter != operator_const_mapping.end();) {
```

Comment:
I would have made this a `while` loop, but that's nitpicking.  I had to do a double-take and reread it because an iterator increment followed by a continue inside a `for` loop is usually a bug.  (I can see that it's not a bug in this case.)

Reply:
Unfortunately, there is no `remove_if` for maps so there isn't a better way for expressing this yet.